docs/how-to-guides/install-seven-add-on.md

```diff
@@ -0,0 +1,189 @@
+---
+myst:
+  html_meta:
+    "description": "How to install a Seven frontend add-on with Cookieplone"
+    "property=og:description": "How to install a Seven frontend add-on with Cookieplone"
+    "property=og:title": "Install Seven with Cookieplone"
+    "keywords": "Plone, Plone 6, Volto, create, project, install, Cookieplone"
+---
+
+
+(install-cookieplone-label)=
+
+# Install Seven with Cookieplone
+
+This chapter describes how you can create a web application using the {term}`Cookieplone` template.
+
+This template is the recommended way to start a new add-on using the Seven frontend.
```

Comment:
The heading and its following two sentences are not clear. Will this procedure install Seven, create an add-on or project using Seven, or both?

Reply:
Ugh, and the meta stuff says a third option, "install a Seven frontend add-on with Cookieplone". We need to keep this story clear, not muddle it.